models/articles.models.js

```diff
@@ -16,9 +16,30 @@ exports.retrieveArticleById = (articleId) => {
   });
 };
 
-exports.findArticles = () => {
-  const queryString = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(*) AS comment_count 
-  FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC`;
+exports.findArticles = (sort_by = "created_at", order = "desc") => {
+  let queryString = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(*) AS comment_count 
```

Comment:
Good job with the count - if you would like to alter the data type of this value then have a look at casting. You could alter it to be a number rather than a string